runtime/resolvers/resolvers_test.go

```diff
@@ -6,185 +6,248 @@ import (
 	"encoding/csv"
 	"encoding/json"
 	"flag"
-	"maps"
 	"os"
 	"path/filepath"
-	"regexp"
-	"strconv"
 	"strings"
 	"testing"
 
+	runtimev1 "github.com/rilldata/rill/proto/gen/rill/runtime/v1"
 	"github.com/rilldata/rill/runtime"
+	"github.com/rilldata/rill/runtime/pkg/fileutil"
 	"github.com/rilldata/rill/runtime/testruntime"
 	"github.com/stretchr/testify/require"
+	"golang.org/x/exp/maps"
 	"gopkg.in/yaml.v3"
 )
 
-type Resolvers struct {
-	Project    map[string]yaml.Node
-	Connectors map[string]*testruntime.InstanceOptionsForResolvers
-	Tests      map[string]*Test
+// TestFileYAML is the structure of a test file.
+// One runtime instance will created for each test file, with the provided file contents and connectors initialized.
+// Each test in the file will be run in sequence against that runtime instance.
+// The available connectors are defined in runtime/testruntime/connectors.go.
+type TestFileYAML struct {
+	Connectors []string             `yaml:"connectors,omitempty"`
+	Files      map[string]yaml.Node `yaml:"files"`
```

Comment:
Can this be called `Resources` instead of `Files`? I got somewhat confused between test file vs this internal file while reading `ReadMe` and calling this resource would prevent confusion.